.github/workflows/autocorrect-lint.yml

```diff
@@ -1,44 +0,0 @@
-# This workflow to use AutoCorrect tool for checking the copywriting, correct spaces and punctuations for CJK contents.
-#
-# For example:
-#
-# - incorrect: "欢迎阅读MDN文档."
-# - correct:   "欢迎阅读 MDN 文档。"
-#
-# - incorrect: "Welcome，this is MDN Web Docs。"
-# - correct:   "Welcome, to read MDN Web Docs."
-#
-# More details:
-# https://github.com/huacnlee/autocorrect
```

Comment:
It might be nice to have this documented somewhere in this repo, but not a blocker for this PR.